### PR TITLE
[MIG] account_move_report: Migrate account_move_report to 15.0 t#54296

### PR DESCRIPTION
**Pull request by ajqn9094**

### CODE REVIEW

**Comment by ajqn9094** [comment]
@luisg123v Could you review please?

Thanks in advance!

**Review by luisg123v — CHANGES_REQUESTED** [review]
Please keep commit history, you can follow instructions depicted [in the OCA](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0#technical-method-to-migrate-a-module-from-140-to-150-branch)

**Comment by ajqn9094** [comment]
> Please keep commit history, you can follow instructions depicted [in the OCA](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0#technical-method-to-migrate-a-module-from-140-to-150-branch)

On it, thanks!

**Comment by ajqn9094** [comment]
@luisg123v command applied to get the historical from account_move_report

**Review by desdelinux — APPROVED** [review]
LGTM @luisg123v @ajqn9094

File: account_move_report/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "version": "15.0.1.0.0",
+    "author": "Vauxoo",
+    "category": "Accounting/Accounting",
+    "website": "http://www.vauxoo.com/",
+    "license": "LGPL-3",
+    "depends": [
+        "account",
+    ],
+    "demo": [],
+    "data": ["views/account_reports.xml", "views/account_move_report.xml"],
```
Comment (luisg123v) [inline account_move_report/__manifest__.py:12]: Trailing comma, this was already fixed in v14

**Review by luisg123v — CHANGES_REQUESTED** [review]
- Missing black commit from v14. Besides, already fixed issues are not fixed here yet. You should bring history again.
- Title: remove task ID
- Title: remove module name from short description (title) as it's redundant. It should be: `[MIG] module_name: migrate to 15.0`

**Comment by hugho-ad** [comment]
why did you not promote use https://git.vauxoo.com/vauxoo/mexico/-/tree/15.0/l10n_mx_edi_account_move_report?

I think they are almost the same

**Comment by luisg123v** [comment]
@desdelinux could you review @hugho-ad's proposal, please?

**Comment by desdelinux** [comment]
> why did you not promote use https://git.vauxoo.com/vauxoo/mexico/-/tree/15.0/l10n_mx_edi_account_move_report?
> 
> I think they are almost the same



> [l10n_mx_edi_account_move_report](https://git.vauxoo.com/vauxoo/mexico/-/tree/15.0/l10n_mx_edi_account_move_report)?

l10n_mx_edi_account_move_report contains fields that only apply to Mexico.

In multi-country instances, this may not be expected.

cc @luistorresm @hbto could you review @hugho-ad proposal, please?

**Comment by desdelinux** [comment]
@luistorresm Recommend that we at vauxoo use l10n_mx_edi_account_move_report for auditing.

So let's proceed that way.

@luisg123v What other client do you have mapped that use account_move_report?

cc @hugho-ad

**Comment by luisg123v** [comment]
> @luisg123v What other client do you have mapped that use account_move_report?

None that I'm aware of

@ajqn9094, please replace `account_move_report` by `l10n_mx_edi_account_move_report` in v14 and let's forward-port the change to it to v15.

**Pull request closed without merging**